configs/ASRock/X370-Gaming-K4.conf

```diff
@@ -0,0 +1,115 @@
+chip "nct6779-isa-*"
+
+    # Fans
+    label fan1 "Chassis Fan 3 Speed"
+    label fan2 "CPU Fan 1 Speed"
+    label fan3 "CPU Fan 2 Speed"
+    label fan4 "Chassis Fan 1 Speed"
+    label fan5 "Chassis Fan 2 Speed"
+
+    # Temperatures
+    label temp1 "Motherboard"
+    label temp2 "CPU MB"
+    label temp3 "Auxillary"
+    label temp4 "VRM"
+    label temp5 "AUXTIN2"
+    label temp6 "CPU (PECI)" #Almsot always at -25. Not certain what to do about this.
+    
+    # A Thermal BIAS value generated by CPU that includes an offset to drive mainboard Fans to higher speeds.
+    # This value is rounded off to the nearest ~1C and is only reported if "Use Tctrl" has been selected in UEFI as the fan control sensor.
+    # It will likely be offset by 20/10C on Ryzen X series CPU's
+    label temp7 "CPU tCTL"
+
+    # Voltages
+    
+    # VCore is different to VDDCR_CPU this is a voltage measured somewhere by the Firmware (Readings may appear low at times (0.54V) and fluctuate - this is normal)
+    label in0 "VCore"
+    label in1 "VPPM" #scalar
+    label in2 "AVCC"
+    label in3 "3VCC"
+    label in4 "+12V" #scalar
+    label in5 "VDDCR_SOC" #(Data Fabric)
+    label in6 "DRAM (VDDIO)"
+    label in7 "3VSB"
+    label in8 "VBat"
+    label in9 "Null"
```

Comment:
What is "Null"?

Reply:
Null is an unused input. It's not reported in the UEFI and is always at 0.

Reply:
I'd consider using `ignore` then, or perhaps labeling it as "unused".